mobile/Makefile

```diff
@@ -20,7 +20,14 @@ status-desktop-rcc: clean-status-desktop-rcc $(STATUS_DESKTOP_RCC)
 
 $(STATUS_GO_LIB): $(STATUS_GO_FILES) $(STATUS_GO_SCRIPT)
 	@echo "Building Status Go"
+ifeq ($(OS),ios)
+	@STATUS_GO=$(STATUS_GO) \
+		CC="/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang" \
+		CGO_CFLAGS="-arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS18.0.sdk -miphoneos-version-min=16.0 -fembed-bitcode" \
+		$(STATUS_GO_SCRIPT) $(HANDLE_OUTPUT)
+else
```

Comment:
```suggestion
ifeq ($(OS),ios)
	@STATUS_GO=$(STATUS_GO) \
		CC="$(shell xcrun --sdk ${SDK} --find clang)" \
		CGO_CFLAGS="-arch ${ARCH} -isysroot $(shell xcrun --sdk ${IPHONE_SDK} --show-sdk-path) -miphoneos-version-min=${IOS_TARGET} -fembed-bitcode" \
		$(STATUS_GO_SCRIPT) $(HANDLE_OUTPUT)
else
```

Reply:
we'll need to preserve the option to run in the simulator and to easily update the IOS target

Reply:
good suggestion